[esx]/es_extended/server/main.lua

```diff
@@ -600,3 +601,18 @@ AddEventHandler('txAdmin:events:scheduledRestart', function(eventData)
 		end)
 	end
 end)
+
+RegisterNetEvent('esx:setDuty')
+AddEventHandler('esx:setDuty', function(bool)
+    local xPlayer = ESX.GetPlayerFromId(source)
+    if xPlayer.job.onDuty == bool then return end
+    
+    if bool then
+        xPlayer.setDuty(false)
+        xPlayer.triggerEvent('esx:showNotification', _U('stopped_duty'))
+    else
+        xPlayer.setDuty(true)
+        xPlayer.triggerEvent('esx:showNotification', _U('started_duty'))
+    end
```

Comment:
This is false since people will use it in this way: `esx:setDuty, true` so `bool` will be true. Code should be replace by:
```lua
if bool then
        xPlayer.setDuty(true)
        xPlayer.triggerEvent('esx:showNotification', _U('started_duty'))
else
        xPlayer.setDuty(false)
        xPlayer.triggerEvent('esx:showNotification', _U('stopped_duty'))
end